### DISCUSSION TITLE
Which version of Yarn should I use?

### DISCUSSION THREAD

**Discussion by ddenev** [quasarframework/quasar#8928] [General - Components / Directives / etc] [answered]
Hi,

Which version of Yarn (v1 or v3) is the recommended to use with Quasar? Docs do not mention Yarn version anywhere.

**Comment by metalsadman** [comment]
haven't tried v3, but 1.x works fine.

**Comment by rstoenescu** [comment, marked as answer]
I think you meant Yarn v1 or v2. There is no v3. Go with yarn v1.

**Reply by chrisvfritz** [reply]
Is this recommendation still valid? I'm on Yarn 3.6.1 with `nodeLinker: node-modules` and so far have not experienced any issues with Quasar.
